go/worker/keymanager/service.go

```diff
@@ -0,0 +1,32 @@
+package keymanager
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/oasislabs/oasis-core/go/common"
+	"github.com/oasislabs/oasis-core/go/common/accessctl"
+	keymanager "github.com/oasislabs/oasis-core/go/keymanager/api"
+	"github.com/oasislabs/oasis-core/go/runtime/enclaverpc/api"
+)
+
+var _ api.Transport = (*Worker)(nil)
+
+// CallEnclave sends the request bytes to the target enclave.
+func (w *Worker) CallEnclave(ctx context.Context, request *api.CallEnclaveRequest) ([]byte, error) {
+	if mustAllow := w.mustAllowAccess(ctx, request.Payload); !mustAllow {
+		// TODO: In the future we should be using the namespace type for runtime IDs.
```

Comment:
So this is unchanged from before, what this comment is referring to is using `common.Namespace` as the runtime ID type instead of `signature.PublicKey` which is used now. The reason is that runtime IDs are not really public keys (see my comment in https://github.com/oasislabs/oasis-core/issues/1693#issuecomment-506937420).